apps/base/traefik-hub/release.yaml

```diff
@@ -24,6 +24,9 @@ spec:
       - --hub.metrics.opentelemetry.insecure
       - --hub.metrics.opentelemetry.address=prometheus.monitoring:9090
       - --hub.metrics.opentelemetry.path=/api/v1/otlp/v1/metrics
+    providers:
+      ingressClass:
+        enabled: false
```

Comment:
Remove these lines once https://github.com/traefik/traefik-hub-helm-chart/pull/66 is released

```suggestion

```